packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -150,7 +150,7 @@ export default class WalletService {
   }
 
   public get = (id: string) => {
-    if (id === undefined) throw new Error(i18n.t('is-required', { field: 'id' }))
+    if (id === undefined) throw new Error(i18n.t('messages.is-required', { field: 'id' }))
```

Comment:
Under what circumstance would we pass in `undefined` id?

Reply:
When the neuron-ui has bugs that fetching a wallet with `id=undefined`

Reply:
Merging this. But is there any way we can prevent this kind of UI bug from happening?